client/src/Components/Student.module.css

```diff
@@ -0,0 +1,232 @@
+.head{
+    left: -8px;
+    font-family:Arial, Helvetica, sans-serif;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 45px;
+    text-align: left;
+    color: #FFFFFF;
+    background-color: black;
+    padding: 20px;
+    margin-bottom: 0px;
+    margin-top: 0px;
+}
+.signout{
+    position: absolute;
```

Comment:
We want our page layout to be dynamic. Here you can used `position: absolute` which means the page will look broken at most screen sizes. Please have a look at the following link:

https://css-tricks.com/dont-overthink-flexbox-grids/

and update the CSS so that it produces dynamic layouts